integrated_channels/sap_success_factors/exporters/course_metadata.py

```diff
@@ -109,13 +110,20 @@ def transform_status(self, course_run):
             else self.STATUS_INACTIVE
         )
 
-    def transform_title(self, course_run):
```

Comment:
is there a reason this function name changed? transform_title is consistent with what is in degreed above

Reply:
Name didn't change but I pulled out a bulk of it to be reused in the `transform_content` method as well. I like Doug's suggestion of `format_title` to be a bit more descriptive than `get_title`